index.html

```diff
@@ -370,9 +370,15 @@ <h3>Ecosystem Overview</h3>
           <dd>
 A role an [=entity=] performs by asserting [=claims=] about one or
 more [=subjects=], creating a [=verifiable credential=] from these
-[=claims=], and transmitting the [=verifiable credential=] to a
-[=holder=]. Example issuers include corporations, non-profit organizations,
+[=claims=], and transmitting the [=verifiable credential=] to the
+[=issuee=]. Example issuers include corporations, non-profit organizations,
 trade associations, governments, and individuals.
+          </dd>
+          <dt>[=issuee=]</dt>
+          <dd>
+A role an [=entity=] might perform by receiving one or more
+[=verifiable credentials=] from an [=issuer=]. An issuee is a subclass
+of the holder role.
```

Comment:
```suggestion
[=verifiable credentials=] directly from an [=issuer=]. The issuee
role is a subclass of the [=holder=] role.
```

Reply:
The word "directly" is not in the current text so I suggest not to make this change. 
I accept your changes to the role sentence, thanks.

Reply:
The word "directly" is what ensures that the `issuee` does not receive the VC from an intermediary `holder` ... Or is such a relay possible in your mental model? Is `issuee` defined simply as "the entity identified as such within the VC", rather than reflecting anything about the VC's passage from `issuer` to (`holder` to) `issuee`??

Reply:
Maybe an `issuee` is one or more entities that an issuer explicitly identifies as intended holders -- and that's all that needs to be said. I don't expect us to come to consensus on any text that reduces the generality we have today around the flow of VCs, but text that identifies some special case of holder roles on top of it is probably acceptable.